app/src/main/res/layout/dialog_image_preview.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_gravity="bottom">
+
+    <androidx.core.widget.NestedScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:fillViewport="true">
+
+        <LinearLayout
+            android:id="@+id/dialogDetailContainer"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:animateLayoutChanges="true"
+            android:orientation="vertical">
+
+            <LinearLayout
+                android:id="@+id/toolbarView"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="?attr/selectableItemBackground"
+                android:clickable="true"
+                android:minHeight="64dp">
+
+                <ImageView
+                    android:id="@+id/downChevron"
+                    android:layout_width="@dimen/bottom_toolbar_icon_size"
+                    android:layout_height="@dimen/bottom_toolbar_icon_size"
+                    android:layout_gravity="center_vertical"
+                    android:contentDescription="@null"
+                    android:padding="12dp"
+                    android:scaleY="-1"
+                    android:tint="?attr/main_toolbar_icon_color"
+                    app:srcCompat="@drawable/ic_keyboard_arrow_up_black_24dp" />
+
+                <TextView
+                    android:id="@+id/titleText"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_vertical"
+                    android:layout_weight="1"
+                    android:ellipsize="end"
+                    android:fontFamily="serif"
+                    android:lineSpacingMultiplier="0.9"
+                    android:maxLines="3"
+                    android:paddingBottom="4dp"
+                    android:textColor="?attr/primary_text_color"
+                    android:textSize="14sp"
+                    tools:text="Lorem ipsum" />
+            </LinearLayout>
+
+            <org.wikipedia.views.ZoomableDraweeViewWithBackground
+                android:id="@+id/galleryImage"
+                android:layout_width="match_parent"
+                android:layout_height="446dp"
+                android:contentDescription="@null"
+                app:actualImageScaleType="fitXY" />
```

Comment:
Can we use `fitCenter` for the `scaleType`?

Reply:
Yes I tried that since it doesnt distort the image... but I am guessing that the height is fixed in the design and with fitCenter it will not be. So will re-check with Robin once and then change